### PR TITLE
Energy detection implemented

### PR DESCRIPTION
**Pull request by JarkkoPaso**

### CODE REVIEW

**Comment by JarkkoPaso** [comment]
@SeppoTakalo @mikter review. Is this going to right repo/branch?

File: source/driverAtmelRFInterface.cpp [hunk 1]
```diff
@@ -878,6 +878,30 @@ void rf_if_set_channel_register(uint8_t channel)
 }
 
 /*
+ * \brief Function enables RF irq pin interrupts in RF interface.
```
Comment (SeppoTakalo) [inline source/driverAtmelRFInterface.cpp]: Wrong kind of Doxygen tags.
Use
```
/** \brief plaaplaa
 * Joojooo.
 */
```

**Comment by SeppoTakalo** [comment]
+1

File: source/driverAtmelRFInterface.cpp (hunk 1 above)
Comment (JarkkoPaso) [inline source/driverAtmelRFInterface.cpp]: Should we then change them to entire file? Or all source files? Not just these two new functions?

Comment (SeppoTakalo) [inline source/driverAtmelRFInterface.cpp]: Ah ok. leave as is. Need separate commit for fixing those.

**Comment by JarkkoPaso** [comment]
@kjbracey-arm Could you check that I didn't break anything while fixing the conflict.

**Comment by kjbracey-arm** [comment]
Seems fine to me.

**Pull request merged**